Source: David Legrand <david.legrand@clever-cloud.com>
Upstream: Yes (#21095)
Reason: Add `native_test.v` to the list of skippable tests as it uses `ld`
---
 cmd/tools/vtest-self.v | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/tools/vtest-self.v b/cmd/tools/vtest-self.v
index 1c4b2a7df..e374b8c86 100644
--- a/cmd/tools/vtest-self.v
+++ b/cmd/tools/vtest-self.v
@@ -342,6 +342,7 @@ const skip_on_non_amd64_or_arm64 = [
 const skip_on_sandboxed_packaging = [
 	'do_not_remove',
 	'vlib/v/slow_tests/inout/compiler_test.v',
+	'vlib/v/gen/native/tests/native_test.v',
 	'vlib/v/compiler_errors_test.v',
 	'vlib/v/gen/c/coutput_test.v',
 ]
-- 
2.39.3 (Apple Git-146)
